gui/src/main/java/org/jboss/as/console/mbui/widgets/ModelNodeFormBuilder.java

```diff
@@ -649,17 +649,23 @@ public int compare(Property property, Property property1) {
                     // iterate over the "requires" attribute list
 
                     boolean sourceItemDefined = isFormItemDefined(sourceFormItem);
```

Comment:
That was my first attempt at fixing it but it doesn't work:
`boolean itemUndefined = !isFormItemDefined(item);` - this needs to be false when item == null
`boolean fieldIsInUse = isFormItemDefined(sourceFormItem);` - and this also needs to be false
i.e. when checking alternatives you treat "null" as undefined, but then checking requires you need "null" to mean "the value is set" (so that the validation passes).